ddcz/models/used/discussion.py

```diff
@@ -3,6 +3,9 @@
 from ..magic import MisencodedCharField, MisencodedTextField
 from .users import UserProfile
 
+ADD_PHORUM_COMMENT = "a"
```

Comment:
Just for the future, I'd standardize on Enums, I think they are more readable as they wrap choices in their own namespace and they can be compared as constants and not as strings. Approving this one though :)